apps/root/src/services/accountService.ts

```diff
@@ -11,13 +12,13 @@ enum WalletType {
 type Wallet = {
   type: WalletType;
   address: string;
-  label: string;
+  label?: string;
```

Comment:
I believe it would be useful to handle `undefined` values. In the UI, we would easily detect if there is no label set and choose to show or hide the label component. Otherwise, we would be showing the same value duplicated (Label + Address) when no label is set